substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/VMFeature.java

```diff
@@ -89,6 +92,9 @@ public void afterAnalysis(AfterAnalysisAccess access) {
             nativeLibraries.getStaticLibraries().stream().map(current::relativize).map(Path::toString).forEach(x -> System.out.println("#   " + x));
             System.out.println("# Other libraries: " + String.join(",", nativeLibraries.getLibraries()));
         }
+
+        CGlobalData<PointerBase> isStaticBinaryMarker = CGlobalDataFactory.createWord(WordFactory.unsigned(SubstrateOptions.StaticExecutable.hasBeenSet() ? ~0 : 0), STATIC_BINARY_MARKER_SYMBOL_NAME);
```

Comment:
I'd use `-1` instead of `~0` (although a matter of taste).

Reply:
I have changed it to -1 instead of ~0. :)